lib/logstash/inputs/http.rb

```diff
@@ -11,26 +11,44 @@ class Puma::Server
   def normalize_env(env, client); end
 end
 
-# This output opens a web server and listens to HTTP requests,
-# converting them to LogStash::Event instances.
+# Using this input you can receive single or multiline events over http(s).
```

Comment:
Not sure what is a multiline event in this context, a plaintext body with some "\n" in the middle?